engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/ExpressionIdTest.scala

```diff
@@ -38,13 +38,13 @@ class ExpressionIdTest extends InterpreterTest {
     "be correct in applications and method calls" in
     withIdsInstrumenter { instrumenter =>
       val code =
-        """from Builtins import all
+        """from Standard.Builtins import all
           |
           |main = (2-2 == 0).if_then_else (Cons 5 6) 0
           |""".stripMargin
       val meta = new Metadata
-      val id1  = meta.addItem(33, 36)
-      val id2  = meta.addItem(58, 8)
+      val id1  = meta.addItem(42, 36)
+      val id2  = meta.addItem(67, 8)
```

Comment:
It's worrying to me that so many of our tests rely on these locations.
Because, if the tests are being updated it's so extremely tempting to just run them, see what are the new locations and paste them here without checking if they correspond to the right entities.
It's certainly not within scope of this PR to fix that, but just wanted to mention that maybe at some point we could consider taking some other approach, which could maybe be based on identifying the locations by some infix or something more readable than just an integer location.

cc: @iamrecursion

Reply:
> Because, if the tests are being updated it's so extremely tempting to just run them, see what are the new locations and paste them here without checking if they correspond to the right entities.

Would be very tempting, if that was possible... But the only feedback these give you is "the node you want doesn't exist". Nonetheless, the idea of having special markers for metadata is awesome (and I'm pretty sure @mwu-tow already has something like that in the IDE?)